app/views/users/edit.haml

```diff
@@ -10,6 +10,11 @@
       %tr
         %th.sub= f.label :email
         %td{class: cycle('even', 'odd')}= f.text_field :email, placeholder: "Email address", class: 'text'
+      %tr
+        %th.sub Author profile
+        %td{class: cycle('even', 'odd')}
+          %button{ type: 'button', onclick: "window.location.href='#{profile_edit_user_path(current_user)}'" }
+            Edit Profile
```

Comment:
I would use a link here rather than hacking a button to do the same with JavaScript

```suggestion
          = link_to "Edit Profile", profile_edit_user_path(current_user)
```

Reply:
the constellation channel didn't want the link, they thought it was ugly and preferred a button. I see. you're all wrong but I'll allow it 👀